controllers/idle_handlers.go

```diff
@@ -145,7 +145,7 @@ func (r *ImageBasedUpgradeReconciler) cleanup(
 
 	r.Log.Info("Cleaning up Backup CRs")
 	if allRemoved, err := r.BackupRestore.CleanupBackups(ctx); err != nil {
-		handleError(err, "failed to cleanup backups.")
+		handleError(err, err.Error())
 	} else if !allRemoved {
 		err := errors.New("failed to delete all the backup CRs.")
 		handleError(err, err.Error())
```

Comment:
Kind of reads like we are printing the same error twice. The original `handleError(err, "failed to cleanup backups.")` seems better with basically an additional msg? L151 seems to double print too

Reply:
The first arguement is for logging and the second argument is used for status message. The purpose of this story is to show this detailed error in the status message.

Reply:
updated